dev/package-examples/iptables-1.0.4/filebeat/config/input.yml

```diff
@@ -1,25 +1,32 @@
-{{ if eq .input "syslog" }}
-
+{{setEmptyVar input "syslog"}}
```

Comment:
This helper would set a variable to the given value if the variable is not set yet.